aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml

```diff
@@ -167,6 +239,14 @@
             <name>values</name>
             <value lang="fr">Liste##Carrousel</value>
         </parameter>
+        <parameter>
+            <name>values</name>
+            <value lang="en">List##Carrousel</value>
+        </parameter>
+        <parameter>
+            <name>values</name>
+            <value lang="de">List##Carrousel</value>
+        </parameter>
         <parameter>
             <name>default</name>
             <value lang="fr">carousel</value>
```

Comment:
In French: carrousel (two r)

Reply:
In this part, carousel is just the key. Not a label. So I let "carousel"